scripts/mk.js

```diff
@@ -27246,10 +27330,16 @@ function choose(map,rand) {
 				for (i=0;i<choixJoueurs.length;i++) {
 					if (!choixJoueurs[i][7]) {
 						var oTr = document.createElement("tr");
+
+						let nameCell = document.createElement("td");
+						nameCell.innerHTML = choixJoueurs[i][5];
+						nameCell.style.width = (iScreenScale*20) + "px";
```

Comment:
One problem with adding the names is that the table is larger and can overflow (both horizontally and vertically)
I tried a few design changes to reduce the risk and slightly improve the visual, let me know what you think
```suggestion
						var nameDiv = document.createElement("div");
						nameDiv.style.maxWidth = (iScreenScale*20) + "px";
						nameDiv.style.overflow = "hidden";
						nameDiv.style.textOverflow = "ellipsis";
						nameDiv.style.marginLeft = iScreenScale +"px";
						nameDiv.style.marginRight = iScreenScale +"px";
						nameDiv.style.color = "#ccc";
						nameDiv.style.fontSize = "0.8em";
						nameDiv.innerHTML = choixJoueurs[i][5];
						nameCell.appendChild(nameDiv);
```